docs/cookie_authentication.rst

```diff
@@ -0,0 +1,13 @@
+Cookie authentication
+=====================
+
+When a token is requested and ``jwt_cookie`` decorator is set, the response will set the given cookie with the token string::
```

Comment:
What do you think about mentioning here (or somewhere else) that CSRF token should be used when using cookie as transport mechanism for jwt token?